tests/test_image_types.py

```diff
@@ -0,0 +1,346 @@
+import wagtail_factories
+
+from django.test import override_settings
+from test_grapple import BaseGrappleTest, BaseGrappleTestWithIntrospection
+from wagtail import VERSION as WAGTAIL_VERSION
+from wagtail.images import get_image_model
+
+from grapple.types.images import rendition_allowed
+
+
+class ImageTypesTest(BaseGrappleTestWithIntrospection):
+    @classmethod
+    def setUpTestData(cls):
+        cls.image_model = get_image_model()
+        cls.example_image = wagtail_factories.ImageFactory(
+            title="Example Image", file__filename="grapple-test.png"
+        )
+
+    def tearDown(self) -> None:
+        for rendition in self.example_image.renditions.all():
+            rendition.file.delete(False)
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        cls.example_image.file.delete(False)
+
+    def test_properties_on_saved_example_image(self):
+        example_img = self.image_model.objects.first()
+
+        self.assertEqual(example_img.id, self.example_image.id)
+        self.assertEqual(example_img.title, "Example Image")
```

Comment:
Is this testing factory boy functionality? Is it useful to do that?

Reply:
maybe. no.
and at the same time I was limiting the amount of swooping changes in one PR...